seatunnel-ui/src/components/chart/index.ts

```diff
@@ -16,7 +16,9 @@
  */
 
 import { getCurrentInstance, onMounted, onBeforeUnmount, watch } from 'vue'
+import { useThemeStore } from '@/store/theme'
 import { throttle } from 'echarts'
+import { useI18n } from 'vue-i18n'
 import type { Ref } from 'vue'
 import type { ECharts } from 'echarts'
```

Comment:
did we need add echarts license file?

Reply:
I don't need it for the time being. I think it's possible to supplement the license before the release, like ds.

Reply:
I would say no. don't say like DS, we are still in the incubator, I believe that any project will strictly require a license.